pkg/icingadb/db.go

```diff
@@ -373,7 +384,7 @@ func (db *DB) CreateStreamed(ctx context.Context, entities <-chan contracts.Enti
 	sem := db.getSemaphoreForTable(utils.TableName(first))
 	stmt, placeholders := db.BuildInsertStmt(first)
 
-	return db.NamedBulkExec(ctx, stmt, 1<<15/placeholders, sem, forward, nil)
+	return db.NamedBulkExec(ctx, stmt, db.options.MaxPlaceholdersPerStatement/placeholders, sem, forward, nil)
```

Comment:
Could become 0 if a small value is set. There's no documentation on what the required minimum would be and even if it was documented, it potentially would have to be updated when we add columns and so on. So maybe go with something like `max(maxPlaceholders/placeholders, 1)` here and in the other places?